dev/tests/integration/testsuite/Magento/Customer/Model/OptionsTest.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\Customer\Model;
+
+use Magento\Config\Model\Config\Source\Nooptreq;
+use Magento\Customer\Helper\Address;
+use Magento\Framework\App\Config\MutableScopeConfigInterface;
+use Magento\Framework\ObjectManagerInterface;
+use Magento\Store\Model\ScopeInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Test for \Magento\Customer\Model\Options.
+ * @magentoDbIsolation enabled
+ */
+class OptionsTest extends TestCase
+{
+    private const XML_PATH_SUFFIX_SHOW = 'customer/address/suffix_show';
+    private const XML_PATH_SUFFIX_OPTIONS = 'customer/address/suffix_options';
+    private const XML_PATH_PREFIX_SHOW = 'customer/address/prefix_show';
+    private const XML_PATH_PREFIX_OPTIONS = 'customer/address/prefix_options';
+
+    private const STUB_OPTION_PREFIX_NAME = 'prefix';
+    private const STUB_OPTION_SUFFIX_NAME = 'suffix';
+    private const STUB_CONFIG_VALUES = 'v1;v2';
+    private const STUB_CONFIG_VALUES_WITH_BLANK_OPTION = ';v1;v2';
+    private const STUB_EXPECTED_VALUES_WITH_BLANK_OPTION = [' ' => ' ', 'v1' => 'v1', 'v2' => 'v2'];
+    private const STUB_EXPECTED_VALUES = ['v1' => 'v1', 'v2' => 'v2'];
```

Comment:
Such test data is totally unreadable, please inline all constants and if you need some value in two places just create a variable in-place.